config/crd/experimental/policy.networking.k8s.io_clusternetworkpolicies.yaml

```diff
@@ -72,14 +72,20 @@ spec:
                   properties:
                     action:
                       description: |-
-                        Action specifies the effect this rule will have on matching traffic.
-                        Currently the following actions are supported:
-                        Allow: Accepts the selected traffic. No further ClusterNetworkPolicy/
-                        NetworkPolicy rules will be processed for it.
-                        Deny: Drops the selected traffic. No further ClusterNetworkPolicy/
-                        NetworkPolicy rules will be processed for it.
-                        Pass: Skips all further ClusterNetworkPolicy rules in the current tier
-                        for the selected traffic, and passes execution to the next tier.
+                        Action specifies the effect this rule will have on matching
+                        traffic.  Currently the following actions are supported:
+
+                        - Accept: Accepts the selected traffic, allowing it to
+                          egress. No further ClusterNetworkPolicy or NetworkPolicy
+                          rules will be processed.
+
+                        - Deny: Drops the selected traffic. No further
+                          ClusterNetworkPolicy or NetworkPolicy rules will be
+                          processed.
+
+                        - Pass: Skips all further ClusterNetworkPolicy rules in the
+                          current tier for the selected traffic, and passes
+                          evaluation to the next tier.
                       enum:
                       - Allow
```

Comment:
The enum values in the generated files are still wrong...

Reply:
Fixing.